src/index.js

```diff
@@ -2,12 +2,18 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.scss';
 import App from './Components/App/App';
+import { ViewProvider } from './Context/ViewContext'
+import { SettingsProvider } from './Context/SettingsContext'
 import { BrowserRouter as Router } from 'react-router-dom'
 import * as serviceWorker from './serviceWorker';
 
 ReactDOM.render(
   <Router>
-    <App />
+    <SettingsProvider>
```

Comment:
Too weird that we're getting issues with Provider & Router! Looking forward to learning more about this & `useMemo`.